### ISSUE TITLE
ColPali `colbert-embedder`

### ISSUE THREAD

**Issue by rawsh-rubrik** [vespa-engine/vespa#32389]
Are there plans on supporting multimodal colbert models for built in inference?

**Comment by bratseth** [comment]
Yes, that is on our roadmap for this autumn.

**Comment by AmirEstiri** [comment]
Hey Jon, any news on this issue?

**Comment by bratseth** [comment]
We'll get to it this year!

**Comment by simjak** [comment]
hey, any updates here?

**Comment by bratseth** [comment]
... 1st of January 😅

We're still working on it. Should be Real Soon Now.

**Comment by amanalok** [comment]
Greetings Jon, wanted to check for the ETA regarding the multi-modal embeddings support. Also, apart from Colpali, will there be support for other multi-modal embedding models such as ColQwen, Qwen-DSE ?

**Comment by bratseth** [comment]
Yes, we'll support some additional ones, maybe not right away. This will still be some weeks - I can't be more specific sorry.

**Comment by vitalijssilins** [comment]
Hi @bratseth, thank you for your work. Do you happen to have any updates on this?

**Comment by bratseth** [comment]
Sorry, not yet!
